src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -265,6 +267,42 @@ internal abstract partial class AbstractAddImportCodeFixProvider<TSimpleNameSynt
         }
 
         /// <summary>
+        /// We ignore references that are in a directory that contains the names "Packages".
+        /// These directories are most likely the ones produced by NuGet, and we don't want
+        /// to offer to add .dll reference manually for dlls that are part of NuGet packages.
+        /// </summary>
+        private bool IsInPackagesDirectory(PortableExecutableReference reference)
+        {
+            const string Packages = nameof(Packages);
```

Comment:
Why do you use this pattern? You really want to make sure that "Packages" doesn't get renamed as it has meaning.